### DISCUSSION TITLE
Joystick Adapter

### DISCUSSION THREAD

**Discussion by IMSAI-Oracle** [thehighnibble/thehighnibble.github.io#59] [S-132 add-on for IMSAI & Cromemco replicas]
On the Joystick Adapter the Amp connector on the BOM show's to use a 3 Pin Header but on the **THN038 - S-132 (1) - Introduction - an add-on for the [#IMSAI8080](https://www.youtube.com/hashtag/imsai8080) Replica** it shows a 3 Pin Female Header.

My question is which one is correct ?

Thanks,
Lynn M.

**Comment by GrantMeStrength** [comment]
I think it doesn't matter. Are you going to connect something to it? I just have three header pins soldered there, but I'm not sure what it would connect to. An amplifier of some sort I guess, but I have seen no details about that. Whatever it is, you would be building something yourself so the choice of connector would be up to you.

**Comment by ronaldlw** [comment]
I noticed this discrepancy as well, but considering we're not supplied the kind of header shown in the video and the fact that the BOM specifically calls out "cut from 20 pin male header" it seems the male header is what he landed on.
![image](https://github.com/user-attachments/assets/1e988b32-e030-4f16-b243-0d535c094f3d)
